### DISCUSSION TITLE
outline preview overlay the items after enlarge the outline buffer

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvimdev/lspsaga.nvim#1263] [Q&A]
Beginging: <img width="998" alt="image" src="https://github.com/nvimdev/lspsaga.nvim/assets/95092244/d77b394d-2a76-400a-94d8-f340cd7ae87b">

After enlarge the width: <img width="890" alt="image" src="https://github.com/nvimdev/lspsaga.nvim/assets/95092244/760e4d72-2c35-49ce-83ee-52e0d8ffc2a0">

I want to see more content of the outline but after I enlarged the width of the outline buffer, the preview pop window didn't move according to the width of the outline buffer.

**Comment by glepnir** [comment]
open outline then changed width ? no resized for preview .

**Reply by LintaoAmons** [reply]
Yes, resize the width after realised that the current width not able to display all the information

**Reply by glepnir** [reply]
it's a bit messy add resized for preview. maybe you can try increase outline.win_width `

**Reply by LintaoAmons** [reply]
Is there a key to toggle preview?
![image](https://github.com/nvimdev/lspsaga.nvim/assets/95092244/dba74ad2-c569-4283-bbc1-dd6518a12640)

**Reply by glepnir** [reply]
no toggle command but maybe can add one :)
